source/includes/oai-pmh/_root.md

```diff
@@ -16,6 +16,52 @@ not available for download.
 
 `https://zenodo.org/oai2d`
 
+
+## First steps with Sickle
+
+In order to better understand the capabilities of OAI let’s do some harvesting using a tool called ```Sickle```. 
+Let's first install the ```Sickle``` package using ```pip```:
+
+```python
+pip install Sickle
+```
+
+After that you'll be able to import ```Sickle``` and initialize the connection by passing the ```Base URL```.
+
+```python
+>>> from sickle import Sickle
+>>> sickle = Sickle('https://zenodo.org/oai2d')
+```
+
+## Get information about the OAI-PMH API
+
+In order to get information on the OAI-PMH API we can use the module ```Identify()``` from ```Sickle```.
+Identify lets us knwo multiple aprameters such as:
+
+In order to get information on the OAI-PMH API, we can use the ```Identify``` verb from ```Sickle```. ```Identify``` returns information about the repository such as:
+
+* ```adminEmail```
+* ```baseURL```
+* ```deletedRecord```
+* ```earliestDatestamp```
+* ```granularity```
+* ```protocolVersion```
+* ```raw```
+* ```repositoryName```
+* ```xml```
```

Comment:
In general, I would avoid documenting Sickle or OAI-PMH features here (since the Sickle docs do a much better job at that). Let's assume users that are at least partially familiar with the OAI-PMH spec, and just showcase what an experienced user might do.